setup.py

```diff
@@ -0,0 +1,42 @@
+import os
+from setuptools import setup, find_packages
+
+import versioneer
+
+install_requires = ["requests"]
+
+
+setup(
+    name="saturn-client",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
+    maintainer="Saturn Cloud Developers",
+    maintainer_email="dev@saturncloud.io",
+    license="BSD-3-Clause",
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: BSD License",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering",
+        "Topic :: System :: Distributed Computing",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+    ],
+    keywords="saturn cloud client api",
```

Comment:
```suggestion
    keywords="saturn cloud client library",
```

I expect that average data scientist thinks the word "API" means "service" and wouldn't describe a library as being an API. I think "client library" is a more appropriate description that's more likely to be correctly understood.